src/yb/server/default-path-handlers.cc

```diff
@@ -184,10 +184,20 @@ static void MemTrackersHandler(const Webserver::WebRequest& req, Webserver::WebR
   *output << "<table class='table table-striped'>\n";
   *output << "  <tr><th>Id</th><th>Current Consumption</th>"
       "<th>Peak consumption</th><th>Limit</th></tr>\n";
+  
+  int max_depth = INT_MAX;
+  string depth = FindWithDefault(req.parsed_args, "max_depth", "false");
+  if (depth != "false") {
+    max_depth = std::stoi(depth);
+  }
 
   std::vector<MemTrackerData> trackers;
   CollectMemTrackerData(MemTracker::GetRootTracker(), 0, &trackers);
   for (const auto& data : trackers) {
+    // if the data.depth >= max_depth, skip the info
```

Comment:
nit: Capitalize If and . at the end of comment

Reply:
Okay